homeassistant/components/roku/browse_media.py

```diff
@@ -13,9 +13,14 @@
 
 CONTENT_TYPE_MEDIA_CLASS = {
     MEDIA_TYPE_APP: MEDIA_CLASS_APP,
-    MEDIA_TYPE_APPS: MEDIA_CLASS_DIRECTORY,
+    MEDIA_TYPE_APPS: MEDIA_CLASS_APP,
     MEDIA_TYPE_CHANNEL: MEDIA_CLASS_CHANNEL,
-    MEDIA_TYPE_CHANNELS: MEDIA_CLASS_DIRECTORY,
+    MEDIA_TYPE_CHANNELS: MEDIA_CLASS_CHANNEL,
+}
+
+CONTAINER_TYPES_SPECIFIC_MEDIA_CLASS = {
+    MEDIA_TYPE_APPS: MEDIA_CLASS_APP,
```

Comment:
The container itself should still be directory. Just add its `children_media_class` as done below.